test-suite/misc/coqtop_print-mod-uid.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+export COQBIN=$BIN
+export PATH=$COQBIN:$PATH
+
+TMP=`mktemp -d`
+cd $TMP
+
+function assert_eq() {
+  if [ "$1" != "$2" ]; then
+    echo "coqtop -print-mod-uid" $1 "!=" $2
+    cd /
+    rm -rf $TMP
+    exit 1
+  fi
+}
+
+mkdir theories/
+touch theories/a.v
+coqc -R theories foo.bar theories/a.v
+assert_eq `coqtop -print-mod-uid theories/a.vo` "theories/.coq-native/NFoo_Bar_A"
```

Comment:
Please use $() instead of backquotes (I thought it was a regular string with single quotes and wasconfused until I looked closer)